cmd/restic/cmd_backup.go

```diff
@@ -421,7 +421,16 @@ func findParentSnapshot(ctx context.Context, repo restic.Repository, opts Backup
 }
 
 func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Terminal, args []string) error {
-	err := opts.Check(gopts, args)
+	var cfg interface{}
```

Comment:
If VSSConfig get a !windows counterpart, say `type VSSConfig struct{}`, you should be able to get rid of this interface{}.

Reply:
I see now that it's actually defined in file built for all platforms. Then you don't need an interface{} at all.

Reply:
Fixed in https://github.com/restic/restic/pull/3067/commits/ff7f681722e8c046b29e3819b5ec5b33f48f9ad5